src/database/schemamanager.h

```diff
@@ -1,11 +1,14 @@
-#ifndef SCHEMAMANAGER_H
-#define SCHEMAMANAGER_H
+#pragma once
 
 #include <QSqlDatabase>
 
 #include "preferences/usersettings.h"
 #include "library/dao/settingsdao.h"
 
+/// The SchemaManager reads the database schema from the schemaFile
+/// (res/schema.xml in development) and is responsible for checking
```

Comment:
"in development"? The file is compiled as a resource and read upon startup.

Reply:
I thought the location might be different in the binary. I wanted to make it easy for new developers to find it.